tests/unittest/_torch/modeling/test_multimodal_encoding.py

```
@@ -0,0 +1,572 @@
+        )
+        assert item.item_idx_in_param == -1
+
+    def test_is_frozen(self):
+        item = ModalityItem(
+            src_param_idx=0,
+            item_idx_in_param=0,
+            modality="image",
+            token_count=1,
+            payload={},
+        )
+        with pytest.raises(Exception):  # FrozenInstanceError subclass of AttributeError
```

Ruff (0.15.14)

[warning] Do not assert blind exception: `Exception`

(B017)